README.md

```diff
@@ -117,21 +147,29 @@ with a ***clear description*** and **_illustrative_ screenshots**:
 
 [![create-issue-for-home-screen-icon](https://cloud.githubusercontent.com/assets/24758860/22227322/ae12d9f6-e1c2-11e6-928c-c20cfe8b3912.png)](https://github.com/dwyl/dwyl.github.io/issues/194)
 
-This is a _real_ issue: https://github.com/dwyl/dwyl.github.io/issues/194
-if you want to see how it was _resolved_, click and go check it out!
+> This is a _real_ issue: https://github.com/dwyl/dwyl.github.io/issues/194
+**you _can_ help us _resolve_ it**!
+
+
 
 ### C) Submit the _New Issue_!
 
-+ Once you _submit_ a new issue for a User Story or Issue/Bug a member
+1. Preview the issue _before_ you submit it to ensure any images were uploaded
+and there aren't any issues with the markdown/text in the description.
+
+1. Once you _submit_ a new issue for a User Story or Issue/Bug a member
 of the team (_typically the Product Owner_) will ***review*** the issue.
 
-+ That's _it_ from the side of the _issue_ creator!
+2. That's _it_ from the side of the _issue_ creator! (_thank you!_)
 
 ![thank-you-green-large](https://cloud.githubusercontent.com/assets/194400/22229077/5b0695ee-e1cd-11e6-9001-e5ff53afce36.jpg)
```

Comment:
I still feel like this is too much to scroll past and people won't read any more, but I've already raised it in #58 so we'll see if anyone else feels the same